PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -3993,6 +3993,19 @@ void Finder::addFileNameTitle(const TCHAR * fileName)
 	_pMainMarkings->push_back(EmptySearchResultMarking);
 }
 
+void Finder::setLineNumFormatStr(size_t NumLines)
+{
+	size_t NumDigits = 0;
+	while (NumLines != 0) {

```

Comment:
Bad brace position. Please check contribution guide line for the coding convention of this project.